core/v0/api/core.yaml

```diff
@@ -1488,7 +1488,7 @@ components:
         additional_description:
           $ref: '#/components/schemas/Descriptor'
     Catalog:
-      description: Describes a BPP catalog
+      description: Describes the products or services offered by a BPP. This is typically sent as the response to a search intent from a BAP. The payment terms, offers and terms of fulfillment supported by the BPP can also be included here. The BPP can show hierarchical nature of products/services in its catalog using the parent_category_id in categories. The BPP can also send a ttl (time to live) in the context which is the duration for which a BAP can cache the catalog and use the cached catalog.  <br>This has properties like bbp/descriptor,bbp/categories,bbp/fulfillments,bbp/payments,bbp/offers,bbp/providers and exp<br>This is used in the following situations.<br><ul><li>This is typically used in the discovery stage when the BPP sends the details of the products and services it offers as response to a search intent from the BAP. </li></ul>
```

Comment:
It not just describes the products and services offered by a BPP, it offers much more. So we could define catalog as an object that contains the following information.

1. Providers that have been onboarded by a BPP
2. Products or services offered by each provider
3. Add-ons offered by each provider
4. Promotional / discount offers available
5. Categories of products/services offered by each provider
6. Fulfillment terms offered by each provider
7. Payment terms offered by each provider
8. Locations where the provider's services/products can be accessed/bought